pkg/permissions/permissions_doc.go

```diff
@@ -0,0 +1,105 @@
+package permissions
+
+import (
+	"fmt"
+
+	"github.com/cozy/cozy-stack/pkg/consts"
+	"github.com/cozy/cozy-stack/pkg/couchdb"
+	"github.com/cozy/cozy-stack/pkg/couchdb/mango"
+	"github.com/cozy/cozy-stack/web/jsonapi"
+)
+
+// Permission is a storable object containing a set of rules and
+// several codes
+type Permission struct {
+	pID           string
+	pRev          string
+	ApplicationID string            `json:"application_id"`
+	Permissions   Set               `json:"permissions, omitempty"`
+	ExpiresAt     int               `json:"expires_at, omitempty"`
+	Codes         map[string]string `json:"codes, omitempty"`
```

Comment:
Just a small thing: in the rest of our codebase, we didn't put a space between `omitempty`